utils.ts

```diff
@@ -86,7 +86,8 @@ export function validateInfraState(
 export function getInfraState(environment: EnvironmentConfig): {
   isMaintenanceActive: boolean;
   isDatabaseActive: boolean;
-  areServicesActive: boolean;
+  isUmamiActive: boolean;
+  areGraaspServicesActive: boolean;
   isMigrationActive: boolean;
```

Comment:
I guess for now it's okay, but if we think of adding even more of them, maybe it's easier to read:
```
isActive/active: {
  maintenance: boolean,
  database: boolean,
  umami: boolean,
  ...
}
```